app/helpers/blacklight_helper.rb

```diff
@@ -10,9 +10,11 @@ def document_partial_name(document, base_name = nil)
 
     display_type ||= document.display_type
 
+    display_type ||= document['eds_publication_type']
```

Comment:
@jkeck I tried doing this but the blacklight_helper.rb still needs modification, like so (adding 2nd line):

```
    display_type ||= document.display_type
    display_type ||= document[view_config.display_type_field]
    display_type ||= 'default'
```

and in ArticleController:

```
    config.show.display_type_field = 'eds_publication_type'
```

See https://github.com/projectblacklight/blacklight/blob/f0e814b1b738bd25d3734039a298eb725663386f/app/helpers/blacklight/render_partials_helper_behavior.rb#L112

Maybe the issue is a drift between BL 5.x and 6.x?